scene/gui/control.cpp

```diff
@@ -4236,6 +4350,14 @@ void Control::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::INT, "size_flags_vertical", PROPERTY_HINT_FLAGS, "Fill:1,Expand:2,Shrink Center:4,Shrink End:8"), "set_v_size_flags", "get_v_size_flags");
 	ADD_PROPERTY(PropertyInfo(Variant::FLOAT, "size_flags_stretch_ratio", PROPERTY_HINT_RANGE, "0,20,0.01,or_greater"), "set_stretch_ratio", "get_stretch_ratio");
 
+	ADD_GROUP("Render Transform", "render_");
+	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "render_offset", PROPERTY_HINT_NONE), "set_render_offset", "get_render_offset");
```

Comment:
```suggestion
	ADD_PROPERTY(PropertyInfo(Variant::VECTOR2, "render_offset"), "set_render_offset", "get_render_offset");
```
As per the old suggestion, `PROPERTY_HINT_NONE` is the default here